.circleci/config.yml

```diff
@@ -45,88 +44,38 @@ jobs:
           command: |
             pipenv run flake8 .
       - run:
-          name: Run tests
-          command: |
-            pipenv run coverage run manage.py test --settings=pennlabs.settings.ci
-      - run:
-          name: Upload code coverage
+          name: Run tests & coverage
           command: |
+            pipenv run coverage run manage.py test --settings=pennclubs.settings.ci
```

Comment:
We should keep this general, not specific to pennclubs.

Reply:
indeed